scripts/download_incr_DB.py

```diff
@@ -68,7 +68,7 @@ def getURL():
 	elif AWS_ENDPOINT_URL:
 		return f"{AWS_ENDPOINT_URL}/{BUCKET_NAME}"
 	else:
-		return "http://"+BUCKET_NAME+".s3.amazonaws.com"
+		return "http://"+BUCKET_NAME+".storage.googleapis.com"
```

Comment:
Why? This seems incorrect. The whole point was to also preserve the AWS functionality.

Reply:
@yaron-zilliqa  - if you'd checked the conversation in slack, you'd know why @chetan-zilliqa made this change. @chetan-zilliqa  - if you'd linked this PR to the JIRA issue, @yaron-zilliqa would've known why you did it, and if you'd linked to the slack discussion in the issue, he'd've been able to find the relevant conversation and probably come up with a fix.

Reply:
For reference: https://zilliqa-jira.atlassian.net/jira/software/c/projects/ZIL/issues/ZIL-5453
The ticket to add the support on testnet repo for both CDN and GCP has been added into the backlog https://zilliqa-jira.atlassian.net/jira/software/c/projects/ZIL/issues/ZIL-5455